docs/articles/documentation/reference/command-line-interface.md

```diff
@@ -951,6 +952,18 @@ The `--disable-multiple-windows` option disables support for multi-window testin
 
 *Related configuration file property*: [disableMultipleWindows](configuration-file.md#disablemultiplewindows).
 
+### --retry-test-pages
+
+If this option is enabled, TestCafe registers a Service Worker that retries failed network requests for the starting test page. If a request does not succeed within ten retries, the test fails.
```

Comment:
I propose another scheme for the feature description:
- what feature do
- what browser capabilities the feature requires

Also, there are two incorrect phrases:
* for the starting test page.
* If a request does not succeed within ten retries, the test fails

#### Another description
```js
If this option is enabled, TestCafe retries failed network requests for the visited/navigated HTML pages. 
The maximum retries is 10 times.

Since this feature uses [ServiceWorker](link to mdn) internally, it requires a secured connection. 
To run TestCafe over secure connection setup the [HTTPS] settings or use the [--hostname localhost>](#--hostname-name) option.

```

Reply:
Yes, this way the main idea is cleare